<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 243, 246);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>851</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>491</width>
       <height>551</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 191, 160);</string>
     </property>
     <widget class="QTextEdit" name="textEdit_output">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>471</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 230, 251);</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_input">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>471</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 230, 251);</string>
      </property>
      <property name="documentTitle">
       <string/>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:8.83436pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.83436pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_translate">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>219</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(137, 207, 240) ;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>翻译</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>331</width>
       <height>571</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 227, 150);</string>
     </property>
     <widget class="QPushButton" name="add">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(137, 207, 240) ;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>添加到单词本</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(137, 207, 240) ;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>从单词本删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cet">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(137, 207, 240) ;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>CET6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(137, 207, 240) ;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>快速笔记</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(137, 207, 240) ;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>单词本</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_open">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(137, 207, 240) ;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>导入文档</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_save">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(137, 207, 240) ;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>导出文档</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(137, 207, 240) ;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>10w_words</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_origin">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>10</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;OCR A Extended&quot;;
font: 700 10pt &quot;Microsoft YaHei UI&quot;;</string>
         </property>
         <property name="text">
          <string>源语言</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_target">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;Microsoft YaHei UI&quot;;
</string>
         </property>
         <property name="text">
          <string>目标语言</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>282</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox_origin">
         <property name="styleSheet">
          <string notr="true">background:rgb(137, 207, 240) ;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
         </property>
         <item>
          <property name="text">
           <string>自动</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>中文</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>英语</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>法语</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>德语</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>日语</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>韩语</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>西班牙语</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_change">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 150, 0);</string>
         </property>
         <property name="text">
          <string>交换</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_target">
         <property name="styleSheet">
          <string notr="true">background:rgb(137, 207, 240) ;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
         </property>
         <item>
          <property name="text">
           <string>中文</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>英语</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>法语</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>德语</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>日语</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>韩语</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>西班牙语</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>138</width>
        <height>67</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="styleSheet">
          <string notr="true">font: 700 10pt &quot;Microsoft YaHei UI&quot;;</string>
         </property>
         <property name="text">
          <string>论文模式</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_Level">
           <property name="styleSheet">
            <string notr="true">font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string> 降重等级</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_Level">
           <property name="styleSheet">
            <string notr="true">background:rgb(137, 207, 240) ;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <item>
            <property name="text">
             <string>初级</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>中级</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>高级</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(137, 207, 240) ;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>高频词汇</string>
      </property>
     </widget>
    </widget>
    <zorder>widget_3</zorder>
    <zorder>widget</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>语言</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionChinese"/>
    <addaction name="actionEnglish"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>单词本</string>
    </property>
    <addaction name="actiond"/>
    <addaction name="actiont"/>
    <addaction name="actions"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>文档翻译</string>
    </property>
    <addaction name="actiondaoru"/>
    <addaction name="actiondaorchu"/>
    <addaction name="separator"/>
    <addaction name="actioneditor"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>单词库</string>
    </property>
    <addaction name="actiondanci"/>
    <addaction name="action10w_words"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>账号</string>
    </property>
    <addaction name="actionss"/>
   </widget>
   <addaction name="menu_5"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="actionPrint"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionFont"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="actionAbout"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionChinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actioneditor">
   <property name="text">
    <string>文本编辑器</string>
   </property>
  </action>
  <action name="actiond">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actiont">
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="actions">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actiondaoru">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="actiondaorchu">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="actiondanci">
   <property name="text">
    <string>CET6</string>
   </property>
  </action>
  <action name="action10w_words">
   <property name="text">
    <string>10w_words</string>
   </property>
  </action>
  <action name="actionss">
   <property name="text">
    <string>退出登录</string>
   </property>
  </action>
  <action name="actionzhanghu">
   <property name="text">
    <string>账号</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>new</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/save_as.png</normaloff>:/images/save_as.png</iconset>
   </property>
   <property name="text">
    <string>save_as</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/print.png</normaloff>:/images/print.png</iconset>
   </property>
   <property name="text">
    <string>print</string>
   </property>
   <property name="toolTip">
    <string>打印</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
   <property name="toolTip">
    <string>离开</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>copy</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>cut</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>paste</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/edit_undo.png</normaloff>:/images/edit_undo.png</iconset>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
   <property name="toolTip">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/edit_redo.png</normaloff>:/images/edit_redo.png</iconset>
   </property>
   <property name="text">
    <string>redo</string>
   </property>
   <property name="toolTip">
    <string>重做</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/font.png</normaloff>:/images/font.png</iconset>
   </property>
   <property name="text">
    <string>font</string>
   </property>
   <property name="toolTip">
    <string>字体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
   </property>
   <property name="text">
    <string>italic</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
   </property>
   <property name="text">
    <string>bold</string>
   </property>
   <property name="toolTip">
    <string>粗体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/underline.png</normaloff>:/images/underline.png</iconset>
   </property>
   <property name="text">
    <string>underline</string>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/info.png</normaloff>:/images/info.png</iconset>
   </property>
   <property name="text">
    <string>about</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="notepad.qrc"/>
 </resources>
 <connections/>
</ui>
